Error:Cannot change strategy of configuration ':app:sandboxWearApp' after it has been resolved.

Issue from : https://github.com/OneSignal/OneSignal-Android-SDK/issues/357

### Project Setup
`build.gradle`
```Gradle
CONTAINS HERE
```


`app/build.gradle`
```Gradle
buildscript {
    repositories {
        maven { url 'https://maven.fabric.io/public' }
    }

    dependencies {
        classpath 'io.fabric.tools:gradle:1.+'
    }
}

plugins {
    id 'com.onesignal.androidsdk.onesignal-gradle-plugin' version '0.7.0'
}
apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'

repositories {
    maven { url 'https://maven.google.com' }
}
apply plugin: 'com.android.application'
apply plugin: 'io.fabric'

allprojects {
    repositories {
        jcenter()
        maven {
            url 'http://www.idescout.com/maven/repo/'
        }
        maven { url "http://dl.bintray.com/pt-midtrans/maven" }
        maven { url 'https://maven.fabric.io/public' }
        maven { url 'https://maven.google.com' }
    }
}
android {
    compileSdkVersion 25
    buildToolsVersion "25.0.1"
    useLibrary 'org.apache.http.legacy'
    defaultConfig {
        multiDexEnabled true
        applicationId "compro.a99vr"
        minSdkVersion 19
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
        manifestPlaceholders = [onesignal_app_id               : "90639a02-3b5b-49f9-a1c7-0d3d320e8308",
                                // Project number pulled from dashboard, local value is ignored.
                                onesignal_google_project_number: "REMOTE"]
    }
    buildTypes {
        debug {
            // Disable fabric build ID generation for debug builds
            ext.enableCrashlytics = false
        }
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    // Define Merchant BASE URL and CLIENT KEY for each flavors
    productFlavors {
        sandbox {
            buildConfigField "String", "BASE_URL", "\"https://merchant-url-sandbox.com/\""
            buildConfigField "String", "CLIENT_KEY", "\"VT-CLIENT-sandbox-client-key\""
        }
    }
}


dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
//    compile project(path: ':SearchView')
    compile project(path: ':SwipeableLayout')
    compile project(path: ':bottomsheet')
    // change the version to latest one

    compile files('libs/activation.jar')
    compile files('libs/additionnal.jar')
    compile files('libs/mail.jar')
    compile('com.crashlytics.sdk.android:crashlytics:2.7.1@aar') {
        transitive = true;
    }
    compile 'com.android.support:appcompat-v7:25.3.1'
    compile 'com.android.support:support-v4:25.3.1'
    compile 'com.android.support:design:25.3.1'
    compile 'com.nhaarman.listviewanimations:lib-core:3.1.0@aar'
    compile 'com.nhaarman.listviewanimations:lib-manipulation:3.1.0@aar'
    compile 'com.nhaarman.listviewanimations:lib-core-slh:3.1.0@aar'
    compile 'com.android.support:cardview-v7:25.3.1'
    compile 'com.github.arimorty:floatingsearchview:2.1.1'
    compile 'com.squareup.okhttp:okhttp:2.4.0'
    compile 'com.squareup.picasso:picasso:2.5.2'
    compile 'com.github.chrisbanes:PhotoView:1.2.6'
//    compile 'com.android.support.constraint:constraint-layout:1.0.0-alpha7'
    compile 'com.nineoldandroids:library:2.4.0'
    compile 'me.himanshusoni.quantityview:quantity-view:1.2.0'
    compile 'com.idescout.sql:sqlscout-server:2.0'
    compile 'com.android.volley:volley:1.0.0'
    compile 'com.android.support:support-v13:21.0.3'
    compile 'com.github.bumptech.glide:glide:3.7.0'
    compile 'pl.droidsonroids.gif:android-gif-drawable:1.2.+'
    compile 'com.android.support:recyclerview-v7:21.0.0'
    compile 'com.daimajia.swipelayout:library:1.2.0@aar'
    compile 'com.andreabaccega:android-form-edittext:1.3.+'
    compile 'com.google.android.gms:play-services-maps:8.3.0'
    compile 'com.google.android.gms:play-services:8.3.0'
    compile 'com.midtrans:uikit:1.10.0-SANDBOX'
    compile 'com.miguelcatalan:materialsearchview:1.4.0'
    compile 'com.onesignal:OneSignal:[3.6.5, 3.99.99]'
    testCompile 'junit:junit:4.12'
}

```

### Full Error
```Java
BUILD ERROR :
Error:Cannot change strategy of configuration ':app:sandboxWearApp' after it has been resolved.
```

### Dependency Tree
Run `./gradlew app:dependencies`
```
PS C:\project-cp\99-virtualrace-app> ^C
PS C:\project-cp\99-virtualrace-app> ./gradlew app:dependencies
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details
NDK is missing a "platforms" directory.leLayout
If you are using NDK, verify the ndk.dir is set to a valid NDK directory.  It is currently set to E:\Program\Android\SDK
\ndk-bundle.
If you are not using NDK, unset the NDK variable from ANDROID_NDK_HOME or local.properties to remove this warning.

Incremental java compilation is an incubating feature.
NDK is missing a "platforms" directory.esolving dependencies ':app:archivesCopy'
If you are using NDK, verify the ndk.dir is set to a valid NDK directory.  It is currently set to E:\Program\Android\SDK
\ndk-bundle.
If you are not using NDK, unset the NDK variable from ANDROID_NDK_HOME or local.properties to remove this warning.

NDK is missing a "platforms" directory.g dependencies ':app:compileCopy' > :bottomsheet > Resolving dependencies ':bott
If you are using NDK, verify the ndk.dir is set to a valid NDK directory.  It is currently set to E:\Program\Android\SDK
\ndk-bundle.
If you are not using NDK, unset the NDK variable from ANDROID_NDK_HOME or local.properties to remove this warning.

OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:design overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:cardview-v7 overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-v13 overridden from '21.0.3' to '25.3.1'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '21.0.0' to '25.3.1'
OneSignalProjectPlugin: com.google.android.gms:play-services overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.0.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.0' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:transition overridden from '25.3.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.3.0' to '25.3.1'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.1.0' to '25.3.1'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-appindexing overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-appinvite overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-appstate overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-drive overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-fitness overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-games overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-identity overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-measurement overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-nearby overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-panorama overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-plus overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-safetynet overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-wallet overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-wearable overridden from '8.3.0' to '8.3.0'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.1.1' to '25.3.1'
OneSignalProjectPlugin: com.android.support:design overridden from '23.1.1' to '25.3.1'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '[10.2.1,11.3.0)' to '8.3.0'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '[26.0.0,26.2.0)' to '25.3.1'
OneSignalProjectPlugin: com.android.support:customtabs overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '22.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:mediarouter-v7 overridden from '22.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '24.2.1' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:palette-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tasks overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:cardview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v13 overridden from '21.0.3' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '21.0.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:transition overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.1.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads-lite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-appinvite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-api-phone overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast-framework overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-config overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-awareness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-crash overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-drive overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fido overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database-connection overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-dynamic-links overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-messaging overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fitness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-games overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gass overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-appindexing overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-identity overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-instantapps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-nearby overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-panorama overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-places overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-plus overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-safetynet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-api overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-v4-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tasks overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wallet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wearable overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:customtabs overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:mediarouter-v7 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '24.2.1' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:palette-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:cardview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v13 overridden from '21.0.3' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '21.0.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:transition overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.1.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads-lite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-appinvite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-api-phone overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast-framework overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-config overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-awareness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-crash overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-drive overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fido overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database-connection overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-dynamic-links overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-messaging overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fitness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-games overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gass overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-appindexing overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-identity overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-instantapps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-nearby overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-panorama overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-places overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-plus overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-safetynet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-api overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-v4-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tasks overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wallet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wearable overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:customtabs overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:mediarouter-v7 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '24.2.1' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:palette-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:cardview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v13 overridden from '21.0.3' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '21.0.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:transition overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.1.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads-lite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-appinvite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-api-phone overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast-framework overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-config overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-awareness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-crash overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-drive overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fido overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database-connection overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-dynamic-links overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-messaging overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fitness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-games overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gass overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-appindexing overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-identity overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-instantapps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-nearby overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-panorama overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-places overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-plus overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-safetynet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-api overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-v4-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tasks overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wallet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wearable overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:customtabs overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:mediarouter-v7 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '24.2.1' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:palette-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:cardview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v13 overridden from '21.0.3' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '21.0.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services overridden from '8.3.0' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-annotations overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:animated-vector-drawable overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-media-compat overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-utils overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-fragment overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:transition overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:recyclerview-v7 overridden from '25.3.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.1.0' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-basement overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-ads-lite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-appinvite overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-api-phone overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-auth-base overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast-framework overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-cast overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-config overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-awareness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-crash overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-drive overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fido overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-auth overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database-connection overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-database overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-dynamic-links overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-messaging overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-storage-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-fitness overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-games overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gass overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-appindexing overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-identity overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-iid overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-instantapps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-maps overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.firebase:firebase-analytics-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-nearby overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-panorama overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-places overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-plus overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-safetynet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-api overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tagmanager-v4-impl overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-tasks overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision-common overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-vision overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wallet overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-wearable overridden from '11.2.2' to '11.2.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.android.support:design overridden from '23.1.1' to '25.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-gcm overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.google.android.gms:play-services-location overridden from '[10.2.1,11.3.0)' to '11.2.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:customtabs overridden from '[26.0.0,26.2.0)' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '23.0.1' to '25.+'
OneSignalProjectPlugin: com.android.support:support-v4 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:mediarouter-v7 overridden from '25.2.0' to '25.+'
OneSignalProjectPlugin: com.android.support:support-core-ui overridden from '24.2.1' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.3.0' to '25.+'
OneSignalProjectPlugin: com.android.support:palette-v7 overridden from '25.4.0' to '25.+'
OneSignalProjectPlugin: com.android.support:appcompat-v7 overridden from '25.0.0' to '25.+'

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Cannot change strategy of configuration ':app:sandboxWearApp' after it has been resolved.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 51.058 secs
PS C:\project-cp\99-virtualrace-app>
```

--- a/src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy
+++ b/src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy
@@ -102,9 +102,14 @@ class GradleProjectPlugin implements Plugin<Project> {
     }
 
     static void doResolutionStrategyAndroidPluginV2(Object configuration) {
+        // The Android 3.3 plugin resolves this before we can
+        // Skip it in this case to prevent a build error
+        if (configuration.name.endsWith('WearApp'))
+            return
+
         configuration.resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-            // Doesn't seem to detect the targetSDK on 2.14
-            // doMinimumVersionUpgradeOnDetail(configuration, details)
+            // The Android 2.14.1 plugin doesn't work with doMinimumVersionUpgradeOnDetail
+            //   We are not able to get the targetSDK version in this case.
 
             // Once doMinimumVersionUpgradeOnDetail is fixed this will loop with compileCopy
             //  At this point we can skip this by checking for ending 'Copy' in the config name

--- a/src/test/groovy/com/onesignal/androidsdk/MainTest.groovy
+++ b/src/test/groovy/com/onesignal/androidsdk/MainTest.groovy
@@ -11,30 +11,33 @@ class MainTest extends Specification {
     String buildFileStr
     File buildFile
 
-    def gradleVersions = [
-        '2.14.1': 'com.android.tools.build:gradle:2.2.3',
-        '4.4': 'com.android.tools.build:gradle:3.0.1'
-    ]
-
-    def buildArgumentSets = [
-        '2.14.1': [
-            ['dependencies', '--configuration', 'compile', '--info'],
-            ['dependencies', '--configuration', '_debugCompile', '--info']
-        ],
-        '4.4': [
-            // compile does not work on it's own for tests since we use variant.compileConfiguration
-            ['dependencies', '--configuration', 'compile', '--info'],
-            ['dependencies', '--configuration', 'debugCompileClasspath', '--info'] //  '--stacktrace'
-        ]
-    ]
+    def gradleVersions = [:]
+    def buildArgumentSets = [:]
 
     def defaultBuildParams = [
         compileSdkVersion: 26,
         targetSdkVersion: 26
     ]
 
-    // Before All tests
-    def setup() { }
+    // Before each test
+    def setup() {
+        gradleVersions = [
+            '2.14.1': 'com.android.tools.build:gradle:2.2.3',
+            '4.4': 'com.android.tools.build:gradle:3.0.1'
+        ]
+
+        buildArgumentSets = [
+            '2.14.1': [
+                ['dependencies', '--configuration', 'compile', '--info'],
+                ['dependencies', '--configuration', '_debugCompile', '--info']
+            ],
+            '4.4': [
+                // compile does not work on it's own for tests since we use variant.compileConfiguration
+                ['dependencies', '--configuration', 'compile', '--info'],
+                ['dependencies', '--configuration', 'debugCompileClasspath', '--info'] //  '--stacktrace'
+            ]
+        ]
+    }
 
     def createManifest(String path) {
         def androidManifest = testProjectDir.newFile("${path}/AndroidManifest.xml")
@@ -89,9 +92,12 @@ class MainTest extends Specification {
                     versionCode 1
                     versionName "1.0"
                 }
+                
                 buildTypes {
                     debug { }
                 }
+
+                ${buildSections['androidSectionExtras']}
             }
             
             dependencies {
@@ -338,6 +344,51 @@ class MainTest extends Specification {
             assert it.value.contains('+--- com.google.android.gms:play-services-location:[10.2.1,11.3.0) -> 11.2.2')
             assert it.value.contains('+--- com.android.support:support-v4:[26.0.0,26.2.0) -> 26.1.0 (*)')
             assert it.value.contains('+--- com.android.support:appcompat-v7:25.0.0 -> 26.1.0')
+            assert it.value.contains('\\--- com.android.support:customtabs:[26.0.0,26.2.0) -> 26.1.0')
+        }
+    }
+
+    def "Ensure flavors work on Gradle 3_3 and latest"() {
+        gradleVersions['3.3'] = 'com.android.tools.build:gradle:2.3.3'
+
+        buildArgumentSets.remove('2.14.1')
+
+        buildArgumentSets['3.3'] = [
+            ['dependencies', '--configuration', 'compile', '--info'],
+            ['dependencies', '--configuration', '_sandboxDebugCompile', '--info']
+        ]
+
+        buildArgumentSets['4.4'] = [
+            ['dependencies', '--configuration', 'compile', '--info'],
+            ['dependencies', '--configuration', 'sandboxDebugCompileClasspath', '--info']
+        ]
+
+        def compileLines = """\
+        compile 'com.onesignal:OneSignal:3.6.4'
+        compile 'com.android.support:appcompat-v7:25.0.0'
+        """
+
+        def androidSectionExtras = """\
+        flavorDimensions 'tier'
+        productFlavors {
+            sandbox {
+                dimension 'tier'
+            }
+        }
+        """
+
+        when:
+        def results = runGradleProject([
+            compileLines: compileLines,
+            androidSectionExtras : androidSectionExtras,
+        ])
+
+        then:
+        results.each {
+            assert it.value.contains('+--- com.google.android.gms:play-services-gcm:[10.2.1,11.3.0) -> 11.2.2')
+            assert it.value.contains('+--- com.google.android.gms:play-services-location:[10.2.1,11.3.0) -> 11.2.2')
+            assert it.value.contains('+--- com.android.support:support-v4:[26.0.0,26.2.0) -> 26.1.0 (*)')
+            assert it.value.contains('\\--- com.android.support:appcompat-v7:25.0.0 -> 26.1.0')
             assert it.value.contains('\\--- com.android.support:customtabs:[26.0.0,26.2.0) -> 26.1.0')
         }
     }

